### ISSUE TITLE
Issue on globby version 8.0.1 after 8.0.2 release

### ISSUE THREAD

**Issue by pusaphil** [sindresorhus/globby#106]
We have been successfully using version 8.0.1 of globby in our code for a while now.

Strangely, we started experiencing issues with our code after the 8.0.2 version release.

**Context:** 

We are using Docker and Kubernetes to build and deploy applications.

After rebuilding a previously working image (using 8.0.1) with no changes at all, our new containers start to crash repeatedly and have the error shown below.

```
INFO [2019-01-15T07:15:32.046Z] (31 on some-app-851366117-zcz88): Running workers...
ERROR [2019-01-15T07:15:32.051Z] (31 on some-app-851366117-zcz88): Expected `cwd` to be of type `string` but received type `undefined`
    TypeError: Expected `cwd` to be of type `string` but received type `undefined`
        at module.exports (/some-app/node_modules/dir-glob/index.js:44:25)
        at globDirs (/some-app/node_modules/globby/index.js:58:9)
        at getPattern (/some-app/node_modules/globby/index.js:61:64)
        at Promise.all.globTasks.map.task (/some-app/node_modules/globby/index.js:72:69)
        at Array.map (<anonymous>)
        at module.exports (/some-app/node_modules/globby/index.js:72:41)
        at Workers.bootstrap (/some-app/app/workers/index.js:22:12)
        at Application.bootstrap (/some-app/app/index.js:14:20)
        at module.exports.Promise.props.then.environment (/some-app/main.js:110:22)
        at runCallback (timers.js:810:20)
        at tryOnImmediate (timers.js:768:5)
        at processImmediate [as _immediateCallback] (timers.js:745:5)
    From previous event:
        at Object.<anonymous> (/some-app/main.js:107:4)
        at Module._compile (module.js:653:30)
        at Object.Module._extensions..js (module.js:664:10)
        at Module.load (module.js:566:32)
        at tryModuleLoad (module.js:506:12)
        at Function.Module._load (module.js:498:3)
        at Function.Module.runMain (module.js:694:10)
        at startup (bootstrap_node.js:204:16)
        at bootstrap_node.js:625:3
```

We use globby to crawl for all the subdirectories with a specific file name.

This is our only line of code that uses it:
```    return globby([`${__dirname}/**/worker.js`]).then(workers => {```

Do you have any ideas why using 8.0.1 is now broken?

**Comment by sindresorhus** [comment]
Duplicate of #98

**Comment by sindresorhus** [comment]
tl;dr Upgrade to version 8.0.3 or newer.
